scripts/ci/build.sh

```diff
@@ -52,6 +52,12 @@ for setup_file in $(find src -name 'setup.py'); do
     popd >/dev/null
 done
 
+##############################################
+# copy private packages
+if [ -z ./privates ]; then
+    cp ./privates/*.whl $output_dir
```

Comment:
Privates would not be in `scripts/ci/` as you use `./privates`.
Or will it be run from the correct directory?

Reply:
It will always run from the root since it is consumed by CI. The script also assumes the `artifacts` folder will locate at the current location as well.